testing/acsccid/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+pkgname=acsccid
+pkgver=1.1.4
+pkgrel=0
+pkgdesc="PCSC driver for ACS CCID smart card reader"
+url="https://github.com/acshk/acsccid"
+arch="all"
+license="LGPL"
+depends="pcsc-lite"
+makedepends="pcsc-lite-dev libusb-dev perl flex"
+options="!check"
+source="https://downloads.sourceforge.net/acsccid/acsccid-${pkgver}.tar.bz2"
+builddir="$srcdir/"${pkgname}-${pkgver}
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="${pkgdir}" install
+	install -Dm644 src/92_pcscd_acsccid.rules "$pkgdir/etc/udev/rules.d/92_pcscd_acsccid.rules"
```

Comment:
I'd like a `-udev` subpackage with install_if=udev here personally, if the package doesn't inherently depend on udev.